client/me/purchases/manage-purchase/plan-details/index.jsx

```diff
@@ -67,10 +68,12 @@ class PurchasePlanDetails extends Component {
 		} );
 
 		return (
-			<div>
```

Comment:
👍  I'm good with the CSS implementation.

I do think we could have a conversation about the pros and cons of adding literal space between inline element 🙂